packages/cli/src/concurrency/concurrency-control.service.ts

```diff
@@ -46,7 +47,7 @@ export class ConcurrencyControlService {
 
 		if (this.productionLimit === -1 || config.getEnv('executions.mode') === 'queue') {
 			this.isEnabled = false;
-			this.log('Service disabled');
+			this.logger.debug('Service disabled');
```

Comment:
Not in the scope of this PR but in the future would be nice to fix logs like this. Having a line

```
[executions] Service disabled
```

doesn't really tell much. Like which service?

Reply:
Agree, I'll overhaul many logs here: https://linear.app/n8n/issue/PAY-2017